addon/components/base/bs-form.js

```diff
@@ -261,6 +280,38 @@ export default Component.extend({
         isPresent(model) && isPresent(property)
       );
       set(model, property, value);
+    },
+
+    /**
+     * Submit action yielded in the bs-form template and accessible as `form.submit`
+     *
+     * @method actions.submit
+     * @param {Object} [model] Optional, falls back to model passed to bs-form
+     * @return {Promise} (not yet implemented)
+     * @public
+     */
+    submit(model=null) {
+      if(model) {
+        this.set('model', model);
```

Comment:
Hm, I am nor sure I like this one. Setting `this.model` would be kind of a violation of the DDAU principle, as it has been set by the user, so we would make this a two way binding if we set it here, right? 

Is there a specific use case you want to support by having the model be passed as an argument here?